eq-author/src/components/ContentPickerv2/DestinationPicker/Menu.js

```diff
@@ -104,24 +112,27 @@ const Menu = ({ data, onSelected, isSelected }) => {
       <Column width={56}>
         <ScrollPane>
           <MenuItemList>
-            {tabs[selectedTab].destinations.map((dest) => (
-              <SubMenuItem
-                key={dest.id}
-                aria-selected={isSelected(dest)}
-                onClick={() => onSelected(dest)}
-                tabIndex={0}
-                onKeyUp={(event) =>
-                  (event.key === Enter || event.key === Space) &&
-                  onSelected(dest)
-                }
-              >
-                <MenuItemTitles>
-                  <MenuItemTitle>
-                    <Truncated>{dest.displayName}</Truncated>
-                  </MenuItemTitle>
-                </MenuItemTitles>
-              </SubMenuItem>
-            ))}
+            {tabs[selectedTab].destinations.map(
+              (dest) =>
+                dest.displayEnabled !== false && (
```

Comment:
```suggestion
                displayEnabled && (
```

You could refactor this down.